controllers/bootstrap/init.go

```diff
@@ -1438,12 +1504,6 @@ func (b *Bootstrap) UpdateOpApproval(operatorName string) error {
 		if err := b.Reader.List(ctx, podList, opts...); err != nil {
 			return err
 		}
-		// may need logic check, we only need to restart the pod of cs operator
```

Comment:
Could we remove line 1499-1506 as well? We don't need to list pods since we don't have to delete them anymore.